apps/speed.c

```diff
@@ -1040,33 +1038,31 @@ static int ECDSA_verify_loop(void *args)
 /* ******************************************************************** */
 static long ecdh_c[EC_NUM][1];
 
-static int ECDH_compute_key_loop(void *args)
+static void ECDH_EVP_derive_key(unsigned char *derived_secret,
```

Comment:
This was changed in 3f435aa95820540daf4366f3da8f1c667523b31b.

Now we only have the ECDH_EVP_derive_key_loop function that directly calls the EVP_PKEY_derive function without any intermediate wrapper.